tools/prune_db.py

```diff
@@ -19,15 +20,16 @@
         print("This will remove ALL runs, measurement, CI, carbonDB and hog data from the DB. Continue? (y/N)")
         answer = sys.stdin.readline()
         if answer.strip().lower() == 'y':
-            DB().query('TRUNCATE runs CASCADE')
-            DB().query('TRUNCATE ci_measurements CASCADE')
-            DB().query('TRUNCATE hog_measurements CASCADE')
-            DB().query('TRUNCATE carbondb_energy_data CASCADE')
-            DB().query('TRUNCATE carbondb_energy_data_day CASCADE')
+            tables = ['runs', 'ci_measurements', 'hog_measurements', 'carbondb_data', 'carbondb_data_raw']
```

Comment:
I think it would be a cleaner option to introduce a switch here where you can select 'non-ee' and 'ee' tables.

Basically the pune_db script could have an '--ee' switch to decided then which tables to include

Reply:
Which ee tables are relevant here? (I don't have access to the enterprise version, so I can't it look up)